js/views/conversation_view.js

```diff
@@ -474,14 +474,17 @@
     },
 
     toggleMicrophone() {
-      if (
-        this.$('.send-message').val().length > 0 ||
-        this.fileInput.hasFiles()
-      ) {
-        this.$('.capture-audio').hide();
-      } else {
-        this.$('.capture-audio').show();
-      }
+      // ALWAYS HIDE until we support audio
+      this.$('.capture-audio').hide();
+
+      // if (
+      //   this.$('.send-message').val().length > 0 ||
+      //   this.fileInput.hasFiles()
+      // ) {
+      //   this.$('.capture-audio').hide();
+      // } else {
+      //   this.$('.capture-audio').show();
+      // }
```

Comment:
Minor but I think we should get into the habit of
`/*
code
code
*/`
when commenting out stuff to minimise the git diffs?